body.tex

```diff
@@ -295,8 +303,13 @@ \subsection{Registering}
 \validation
 \begin{itemize}
     \item $\field{P}{org}$ identifies an existing org $O$,
-    \item $\field{P}{id}$ is unique under $O$, between $1$ and $32$
-        bytes long, and valid \textsf{UTF-8}.
+    \item $\field{P}{id}$ is unique under $O$, and complies with the
+        following rules:
+        \begin{itemize}
+            \item Must be between $1$ and $32$ bytes long.
+            \item Must only contain \texttt{a-z}, \texttt{0-9}, `\texttt{-}'
+                and `\texttt{\_}' characters.
```

Comment:
Is it an idea to call these `name` instead of `id`? `id` presumes that a project would be unequivocally identified by it, but that's not the case since `org` is necessary to have the complete "id" of a project.

In the registry we made that distinction, to have the type and values reflecting that logic.

Reply:
Yeah I remember that discussion, will update.

Reply:
Done.